infra/gcp/terraform/k8s-infra-prow-build-trusted/prow-build-trusted/resources/test-pods/test-pods-serviceaccounts.yaml

```diff
@@ -75,6 +75,14 @@ metadata:
     iam.gke.io/gcp-service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod-bak.iam.gserviceaccount.com
   name: k8s-infra-gcr-promoter-bak
   namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: k8s-infra-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+  name: k8s-infra-promoter
```

Comment:
For followup: did you want a `k8s-infra-promoter-bak` account that can write to `k8s-artifacts-prod-bak`? I see you've got the same for gcr above

Reply:
> For followup: did you want a `k8s-infra-promoter-bak` account that can write to `k8s-artifacts-prod-bak`? I see you've got the same for gcr above

Very possibly! I wanted to minimize the accounts until the need arises, but I've added this as an AI in https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/413.